mozilla/blurts-server
# MNTOR-2934/set UPLOAD_SENTRY_SOURCEMAP correctly

# References: 
Jira: MNTOR-2934




# Description

When tags are applied, upload sourcemaps to Sentry using GH Action.

# How to test

We want this only to run for releases, we can verify this in the "Source Maps" settings of Sentry's web UI. 

# Checklist (Definition of Done)
